packages/twenty-server/src/engine/metadata-modules/object-metadata/object-metadata.service.ts

```diff
@@ -268,6 +268,20 @@ export class ObjectMetadataService extends TypeOrmQueryService<ObjectMetadataEnt
               workspaceId: objectMetadataInput.workspaceId,
               defaultValue: 'now',
             },
+            {
+              standardId: CUSTOM_OBJECT_STANDARD_FIELD_IDS.deletedAt,
```

Comment:
I think similarly to createdAt and updatedAt we can simply reuse the deletedAt from BASE_OBJECT_STANDARD_FIELD_IDS. Wdyt?